### ISSUE TITLE
Automate ownership association in a TrackMe builtin feature

### ISSUE THREAD

**Issue by guilhemmarchand** [trackme-limited/trackme-report-issues#14]
From a user:

```
What I’m totally missing (in contrast to my current manual solution): A feature to define a Data Owner for Hosts and/or Data Sources. Could be a simple email address. The reason for that: Besides the Splunk Admin, every Data Owner should be notified when their source does not provide data any more.
In a perfect world, the data owner can open TrackMe or click a link in the alert email to acknowledge/close the alert. But for me personally it would be totally fine to get the notification out to the Data Owner.
I know I can do this with other ways (e.g. use the tags feature or create a lookup table (source/host<>owner)) but I would prefer to have it as a TrackeMe onboard feature!
```

**Comment by MikeAnderson-E16247** [comment]
This would be very useful for us as well.

**Comment by guilhemmarchand** [comment]
Addressed in the release 2.0.57 with the CMDB integration:
https://docs.trackme-solutions.com/admin_guide_cmdb_integration.html
FYI @MikeAnderson-E16247
